Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -25,7 +25,7 @@ YUI.add('ez-objectrelationsloadplugin', function (Y) {
         },
 
         /**
-         * Loads the related contents. Once this is done, it sets the contents in
+         * Loads the related contents. Once this is done, it sets the content structs in
```

Comment:
well, we talked about that this morning. And the conclusion is to do the solution 1 and to add `@since 1.1` to be follow semver.

> 2. for 1.1.0.
remark: to follow semver, that would be 2.0 actually